docs/Developers/Newcomers_Guide.rst

```diff
@@ -140,6 +140,12 @@ to be part of the Newcomer team. The invitation will be sent by mail and you
 will have to accept it to join. If you don't find the invitation, accept it
 `here <https://github.com/coala>`__.
 
+.. note::
+    If you have any questions, ask them in a precise and respectful way that
+    maximizes your chances of getting the answer you're looking for.
+    If you're unsure how to do that, please read: `How To Ask Questions
+    The Smart Way <http://www.catb.org/esr/faqs/smart-questions.html>`_.
```

Comment:
I would maybe put the note after the "Congratulations" paragraph below. What do you think?